gallery/visualtests/nxCombobox.js

```diff
@@ -0,0 +1,234 @@
+          dropdownButtonSelector = `${backendExampleSelector} .nx-dropdown-button`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('i');
+      await getPage().waitForSelector(dropdownButtonSelector);
+    });
+
+    it('looks right', async function() {
+      const [component] = await waitAndGetElements(backendExampleSelector);
+      await checkScreenshot(component, 300, 376);
+    });
+
+    it('passes a11Y checks', a11yTest());
+  });
+
+  it('looks right with long variant', async function() {
+    const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+        dropdownButtonSelector = `${basicExampleSelector} .nx-dropdown-button`,
+        [component, input] = await waitAndGetElements(basicExampleSelector, inputSelector);
+
+    await input.focus();
+    await getPage().keyboard.type('a');
+    await getPage().waitForSelector(dropdownButtonSelector);
+    await checkScreenshot(component, 800, 376);
+  });
+
+  describe('when displaying empty message', function() {
+    beforeEach(async function() {
+      const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+          emptyMessageSelector = `${basicExampleSelector} .nx-combobox__empty-message`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('q');
+      await getPage().waitForSelector(emptyMessageSelector);
+    });
+
+    it('looks right', async function() {
+      const [component] = await waitAndGetElements(basicExampleSelector);
+      await checkScreenshot(component, 800, 88);
+    });
+
+    it('passes a11Y checks', a11yTest());
+  });
+
+  it('looks right with truncated results', async function() {
+    const inputSelector = `${backendExampleSelector} .nx-combobox__input input`,
+        dropdownButtonSelector = `${backendExampleSelector} .nx-dropdown-button`,
+        [component, input] = await waitAndGetElements(backendExampleSelector, inputSelector);
+
+    await input.focus();
+    await getPage().keyboard.type('loo');
+    await getPage().waitForSelector(dropdownButtonSelector);
+    await checkScreenshot(component, 300, 88);
+  });
+
+  describe('Validation', function() {
+    describe('when valid', function() {
+      beforeEach(async function() {
+        const inputSelector = `${nonEmptinessExampleSelector} .nx-combobox__input input`,
+            dropdownButtonSelector = `${nonEmptinessExampleSelector} .nx-dropdown-button`,
+            [input] = await waitAndGetElements(inputSelector);
+
+        await input.focus();
+        await getPage().keyboard.type('a');
+        await getPage().waitForSelector(dropdownButtonSelector);
+      });
+
+      it('has validation styles', async function() {
+        const [component] = await waitAndGetElements(nonEmptinessExampleSelector);
+        await checkScreenshot(component, 150, 376);
```

Comment:
The screenshot for this one appears to be far too tall

Reply:
Updated here e2931bba013701240a2a0472544d95ef7f3459a4.